templates/pytorch/nightly/fs-transformer.libsonnet

```diff
@@ -125,11 +138,41 @@ local tpus = import "../../tpus.libsonnet";
       },
     },
   },
+  local fastconfig = base.Convergence {
+    paramsOverride: {
+      maxEpoch: 25,
+      trainSubset: "train",
+      validSubset: "valid",
+      inputShapes: "256x64 512x32 640x16",
+      logSteps: 200,
+    },
+    command+: [
```

Comment:
does `command+` work w/ `utils.scriptCommand`?

Reply:
Not sure. I would run jsonnet and see if the resulting yaml looks ok

Reply:
It will compile, but it will give the command `['bash', '-c', '<script>', '<other stuff>']`, which will only run the script.

Reply:
I would run this as a one-shot to make sure that it behaves as you expect. See `templates/README.md` for instructions on doing that.